spanner/oc_test.go

```diff
@@ -261,6 +261,57 @@ func TestOCStats_SessionPool_GetSessionTimeoutsCount(t *testing.T) {
 	}
 }
 
+func TestOCStats_GFE_Latency(t *testing.T){
+	te := testutil.NewTestExporter(GFELatencyView)
+	defer te.Unregister()
+
+	GFELatencyOrHeaderMissingCountEnabled = true
+
+	_, client, teardown := setupMockedTestServer(t)
+	defer teardown()
+
+	client.Single().ReadRow(context.Background(), "Users", Key{"alice"}, []string{"email"})
+
+	waitErr := &Error{}
+	waitFor(t, func() error {
+		select {
+		case stat := <-te.Stats:
```

Comment:
This is where the test case is waiting indefinitely, as there will never be any stats that are exported. That is fixed by adding the additional view at the top of the test. However, the stats that will be exported will at this moment be a `GFEHeaderMissingCount` and not a `GFELatency` value.